anomalib/models/fastflow/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""FastFlow Algorithm Implementation."""
+
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from .lightning_model import Fastflow, FastflowLightning
+from .torch_model import FastflowLoss, FastflowModel
+
+__all__ = ["FastflowModel", "FastflowLoss", "FastflowLightning", "Fastflow"]
```

Comment:
Do we want to expose all these classes here in the init? For the other models we only list the `Lightning` class here. I'm fine with either option, but at least we should be consistent.

Reply:
I think it would be good to be able to import them. Especially when we use them as components. We could do the same for the other algos